src/utils/reclient.js

```diff
@@ -0,0 +1,121 @@
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const tar = require('tar');
+
+const { color, fatal } = require('./logging');
+
+const reclientDir = path.resolve(__dirname, '..', '..', 'third_party', 'reclient');
+const reclientTagFile = path.resolve(reclientDir, '.tag');
+const reclientHelperPath = path.resolve(
+  reclientDir,
+  `electron-rbe-credential-helper${process.platform === 'win32' ? '.exe' : ''}`,
+);
+const rbeServiceAddress = 'rbe.notgoma.com:443';
+
+const CREDENTIAL_HELPER_TAG = 'v0.1.0';
```

Comment:
```suggestion
const CREDENTIAL_HELPER_TAG = 'v0.2.0';
```

Newest?

Reply:
The `0.2.0` credential helper didn't publish correctly because of a codesigning issue